addons/account/views/account_move_views.xml

```diff
@@ -423,20 +422,23 @@
                         ('name', 'ilike', self), ('ref', 'ilike', self), ('account_id', 'ilike', self), ('partner_id', 'ilike', self)]"/>
                     <field name="name"/>
                     <field name="ref"/>
-                    <field name="payment_date" string="Due Date"/>
+                    <field name="payment_date" string="Next Payment Date"/>
                     <field name="partner_id"/>
                     <field name="journal_id"/>
                     <field name="currency_id" groups="base.group_multi_currency" invisible="1"/>
                     <field name="company_currency_id" invisible="1"/>
                     <separator/>
                     <filter string="Posted" name="posted" domain="[('parent_state', '=', 'posted')]" help="Posted Journal Items" invisible="1"/>
                     <separator/>
-                    <filter string="Sales" name="sales" domain="[('journal_id.type', '=', 'sale')]" context="{'default_journal_type': 'sale'}"/>
-                    <filter string="Purchases" name="purchases" domain="[('journal_id.type', '=', 'purchase')]" context="{'default_journal_type': 'purchase'}"/>
+                    <filter string="Invoices" name="invoices" domain="[('amount_residual', '&gt;', '0')]" invisible="context.get('journal_type') != 'sale'"/>
+                    <filter string="Credit Notes" name="credit_notes" domain="[('amount_residual', '&lt;', '0')]" invisible="context.get('journal_type') != 'sale'"/>
+                    <filter string="Bills" name="bills" domain="[('amount_residual', '&lt;', '0')]" invisible="context.get('journal_type') != 'purchase'"/>
+                    <filter string="Refunds" name="refunds" domain="[('amount_residual', '&gt;', '0')]" invisible="context.get('journal_type') != 'purchase'"/>
                     <separator/>
                     <filter string="Invoice Date" name="invoice_date" date="invoice_date"/>
-                    <filter string="Due Date" name="payment_date" date="payment_date"/>
+                    <filter string="Next Payment Date" name="payment_date" date="payment_date"/>
```

Comment:
The string should only be changed in the view ? Not the field itself ?

Reply:
You're right, forgot about that